examples/Cargo.toml

```diff
@@ -35,6 +35,7 @@ required-features = ["ballista/standalone"]
 
 [dependencies]
 ballista = { path = "../ballista/client", version = "0.10.0" }
+ballista-core = { path = "../ballista/core", version = "0.10.0" }
```

Comment:
Same... but `[test-dependencies]`... or maybe we need a `datafusion-test` package. I'm curious to hear what others think.